### PR TITLE
Update 3.19.2

### PR DESCRIPTION
**Pull request by Arishamays1**
## ☆Simplejson 3.19.2 Update ☆
[Jira Ticket](https://anaconda.atlassian.net/browse/PKG-5249)
[Upstream](https://github.com/simplejson/simplejson)
# Changes
- Updated version number and `sha256`
- Skip to `python` versions less than 3.8

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,20 +1,21 @@
...
 
 build:
   number: 0
   #trigger 1
   script:
     - export REQUIRE_SPEEDUPS=1  # [not win]
     - set REQUIRE_SPEEDUPS=1  # [win]
-    - {{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv
+    - {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  skip: True # [py<38]
```
Comment (cbouss) [inline recipe/meta.yaml]:
```suggestion
```
Upstream doesn't mention dropping support of earlier python versions.

**Review by cbouss — APPROVED** [review]
LGTM, with a minor suggestion

File: recipe/meta.yaml (hunk 1 above)
Comment (danpetry) [reply recipe/meta.yaml]: yeah, note that these py skips should always be implemented for reasons other than "our CI doesn't build this python version"

**Review by danpetry — APPROVED** [review]

File: recipe/meta.yaml [hunk 2]
```diff
@@ -43,13 +44,12 @@ about:
...
   license_file: LICENSE.txt
   summary: Simple, fast, extensible JSON encoder/decoder for Python
-
   description: |
     simplejson is a simple, fast, complete, correct and extensible
     JSON <https://json.org> encoder and decoder for Python 2.5+ and
     Python 3.3+. It is pure Python code with no dependencies, but includes
     an optional C extension for a serious speed boost.
-  doc_url: https://simplejson.readthedocs.io/
+  doc_url: https://readthedocs.org/projects/simplejson/
```
Comment (JeanChristopheMorinPerso) [inline recipe/meta.yaml]:
```suggestion
  doc_url: https://simplejson.readthedocs.io/
```
I would keep the previous link.

**Review by JeanChristopheMorinPerso — CHANGES_REQUESTED** [review]

File: recipe/meta.yaml (hunk 2 above)
Comment (Arishamays1) [reply recipe/meta.yaml]: Done thanks! The Linter God isn't always so kind 😂

**Pull request merged**